Dockerfile

```
@@ -23,7 +23,7 @@ ARG PLUMBER_REF=master
 RUN Rscript -e "remotes::install_github('rstudio/plumber@${PLUMBER_REF}')"
 
 EXPOSE 8000
-ENTRYPOINT ["R", "-e", "pr <- plumber::plumb(rev(commandArgs())[1]); pr$run(host='0.0.0.0', port=8000, swagger=TRUE)"]
+ENTRYPOINT ["R", "-e", "pr <- plumber::plumb(rev(commandArgs())[1]); pr$setDocs(TRUE); pr$run(host='0.0.0.0', port=8000)"]
```

Maybe we could check the package version, then we can accommodate both for now. Don't want to disable legacy support if we can get around it with a few characters. Once it starts to look like a hack, we can disable it.

```suggestion
ENTRYPOINT ["R", "-e", "pr <- plumber::plumb(rev(commandArgs())[1]); args <- list(host = '0.0.0.0', port = 8000); if (packageVersion('plumber') >= 1.0.0) { pr$setDocs(TRUE) } else { args$swagger <- TRUE }; do.call(pr$run, args)"]
```

(untested)